### DISCUSSION TITLE
How to handle test failure report and should I report bug each time

### DISCUSSION THREAD

**Discussion by hsandt** [gfx-rs/wgpu#3388] [Q&A] [answered]
Having a graphical issue with Bevy, I followed their recommendation to test wgpu first, which I did, and had 1 failing test with my integrated graphics (I could not test my discrete GPU, but that's another issue), which may be the cause for the issue in Bevy.

Should I just report the test failure and indicate my graphical setup? I haven't found similar issues opened so I don't know if that's a thing often done. More precisely, is there a range of integrated and discrete graphical cards that the project aims at supporting, so I know if my card is considered "too old" and not worth reporting the bug? (of course I'd rather have it supported if possible!)

Are there other extra steps I can take to make sure that my bug report is legit?

**Comment by nical** [comment, marked as answer]
> Should I just report the test failure and indicate my graphical setup? I haven't found similar issues opened so I don't know if that's a thing often done.

Yes, that would be valuable.

> More precisely, is there a range of integrated and discrete graphical cards that the project aims at supporting, so I know if my card is considered "too old" and not worth reporting the bug?

Ideally wgpu would support as large a panel of hardware and driver configurations as possible, but then it's also a matter of whether someone has the time, motivation and hardware available to do the work to fix hardware specific issues. I don't think the project will turn down a good pull request that makes things work on older hardware even if the main wgpu contributors don't have that particular hardware towards the top of their priority list.

In any case it doesn't hurt to have issues on file to know about existence of the problems, should someone eventually find the drive to work on them, and if you feel like contributing a fix, you are more than welcome to as well.

**Reply by Elabajaba** [reply]
Imo there's also the question of does their hardware+driver combo work, and Haswell iGPU + Vulkan (+Linux) is not something that really works, and afaik is something the hardware fundamentally can't fully support.

edit: Hardware info from https://github.com/bevyengine/bevy/issues/7232

**Reply by hsandt** [reply]
Thx, I opened my issue here: https://github.com/gfx-rs/wgpu/issues/3422

It may or may not be related to the bug I see in bevy, so I'll do more specific bevy testing too. I'll also try to run on my discrete GPU.
